package/base-files/files/lib/upgrade/nand.sh

```diff
@@ -10,6 +10,9 @@ CI_KERNPART="${CI_KERNPART:-kernel}"
 # 'ubi' partition on NAND contains UBI
 CI_UBIPART="${CI_UBIPART:-ubi}"
 
+# 'rootfs' partition on NAND contains the rootfs
```

Comment:
Could you please separate out the CI_ROOTPART changes in nand.sh as a separate patch?

Reply:
OK, I'll do it.